deployment/ansible/roles/raster-foundry.app/tasks/static-files.yml

```diff
@@ -41,7 +41,7 @@
 
 - name: Run Django collectstatic command
   django_manage: command=collectstatic app_path="{{ django_home }}"
-  environment: django_config
+  environment: "{{ envdir_config | combine(django_config) }}"
```

Comment:
Did something cause you to have to combine these? Asking to see if I can reproduce the fix.